baseplate/clients/sqlalchemy.py

```diff
@@ -214,7 +214,7 @@ def __init__(self, engine: Engine, name: str = "sqlalchemy"):
         if isinstance(pool, QueuePool):
             self.max_connections_gauge.labels(name).set_function(pool.size)
             self.checked_out_connections_gauge.labels(name).set_function(
```

Comment:
set_function doesn't work in multiprocess mode, why was this reintroduced?

Reply:
Probably the fact we had to rebase, re-create PRs etc. etc.
It was a mistake and should be removed.